### DISCUSSION TITLE
UniDock features video

### DISCUSSION THREAD

**Discussion by npolyak** [AvaloniaUI/Avalonia#6874] [General]
https://www.youtube.com/watch?v=BSEJRS4KyTA

**Comment by robloo** [comment]
Awesome control -- looks like a lot of work too. Great job on this!

**Reply by npolyak** [reply]
Thanks robloo. It took me about two months working after hours and weekends :)

**Reply by npolyak** [reply]
And BTW, please, vote for my video and for my article at [UniDock Power Features.](https://www.codeproject.com/Articles/5316702/UniDock-A-New-Multiplatform-UI-Docking-Framework-U)

**Reply by robloo** [reply]
Wow, really great article as well! This is all really cool to see :)

**Reply by npolyak** [reply]
thanks!
